catalogue/webapp/test/requesting.test.js

```diff
@@ -0,0 +1,21 @@
+const cognitoUrl = 'https://id.wellcomecollection.org/oauth2';
+
+describe('logged out', () => {
+  beforeAll(async () => {
+    await page.setCookie({
+      name: 'toggle_stacksRequestService',
+      value: 'true',
+      domain: 'localhost',
+      path: '/',
+    });
+    await page.goto('http://localhost:3000/works/h6tcdtrt');
+    await page.waitFor(`a[href^="${cognitoUrl}"]`);
+  });
+
+  it('contains a login link', async () => {
+    const loginLink = await page.$(`a[href^="${cognitoUrl}"]`);
+    const linkText = await page.evaluate(link => link.innerHTML, loginLink);
+
+    await expect(linkText).toBe('Login to request and view in the library');
+  });
+});
```

Comment:
The name request is going to get overloaded pretty quick, any reason not to go with stacks?

Reply:
This is what the link text currently says (for better or worse).

Here, 'request and view' are things the user can do – not saying anything about the stacks service.

Or did you mean something else?

Reply:
Na, `requesting.test`. I would call it `stacks.test` and in the `stacks` file, use the test names to mention requesting. `requesting an item works`.

We went round the houses on this one and requesting just felt too generic. It's the stacks service, and requesting an item is part of it.